### PR TITLE
Improve clarity around bug/feature requests, and reorder sections

### PR DESCRIPTION
**Pull request by ahrkrak**
Improved text under reporting bugs, to focus on what to do to raise an issue, and clarify the point about new packages just being an example of a new feature request. I am also proposing reordering the sections to group the key end-user focused sections (report bugs, chat, discussions, mailing lists) before the developer-oriented ones.

# improved readme for key end-user sections (raising bugs, getting in touch with the team)

## How to use

Read the md

## Testing done

Read it in the github browser.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,9 +29,29 @@ The getting started guide has further links to the topics Ignition, local testin
...
-Please see [adding new packages to the Flatcar Linux OS image](adding-new-packages.md) for general guidelines.
+If you hit a bug, or have a feature request, please [file an issue](https://github.com/flatcar/Flatcar/issues/new/choose) right here in this github project.
+Please select the appropriate issue type to help us triage incoming requests. For example, if you would like a new package to be added to the base Flatcar image, use the "New Package Request" issue type. (In that specific case, please also see [adding new packages to the Flatcar Linux OS image](adding-new-packages.md) for general guidelines.)
+
+### Chat
+
+For quick questions or for just hanging out with the community please use
+* Our matrix chat (via element.io): [https://app.element.io/#/room/#flatcar:matrix.org](https://app.element.io/#/room/#flatcar:matrix.org)
+* Our Slack channel in the Kubernetes Slack org: https://kubernetes.slack.com/archives/C03GQ8B5XNJ
+* A bridged IRC channel (Libera.chat): #[flatcar](ircs://irc.libera.chat:6697/#flatcar) channel is also available.
```
Comment (tormath1) [inline README.md]: To be confirmed with @vbatts but IIRC we stopped investigating on this bridge. 
```suggestion
```

Comment (vbatts) [reply README.md]: correct! The bridge is defunct. 
Curiously there are about 13 users sitting in libera.chat #flatcar currently ...

Comment (tormath1) [reply README.md]: Yeah maybe some automatic connections. I was joining automatically `flatcar` channel when I was starting my IRC client for example.

Comment (ahrkrak) [reply README.md]: OK, I had only moved that within this PR. Will delete it.

**Review by tormath1 — APPROVED** [review]

**Review by ader1990 — APPROVED** [review]

**Comment by t-lo** [comment]
Merging PR as it's been approved a while ago.

**Pull request merged**